backend/curriculum_tracking/tests/test_AgileCard.py

```diff
@@ -706,6 +707,111 @@ def test_cancel_workshop_attendance(self):
         self.assertEqual(self.card.status, AgileCard.READY)
 
 
+class ProjectMovementTestCase(TestCase):
+    def setUp(self):
+        
+        self.card = factories.AgileCardFactory(
+            content_item=factories.ContentItemFactory(
+                content_type=ContentItem.PROJECT,
+                project_submission_type=ContentItem.LINK,
+                ),
+            status=AgileCard.READY,
+        )
+        self.card.assignees.set([UserFactory()])
+
+    def test_start(self):
+        card = self.card
+        card.start_project()
+        # self.assertEqual(card.recruit_project.recruit_users, card.assignees.first())
+        card_assignees = sorted([o.id for o in card.assignees.all()])
+        project_assignees = sorted([o.id for o in card.recruit_project.recruit_users.all()])
+
+        self.assertEqual(card.status, AgileCard.IN_PROGRESS)
+        self.assertEqual(card_assignees, project_assignees)
```

Comment:
This all mostly seems to be covered already in this other test class:

```
class start_project_Tests(TestCase):
    def setUp(self):
        bot = SocialProfileFactory(github_name=GIT_REAL_BOT_USERNAME)
        GithubOAuthTokenFactory(user=bot.user)

    def assert_users_same(self, card, project):
        card_assignees = sorted([o.id for o in card.assignees.all()])
        card_reviewers = sorted([o.id for o in card.reviewers.all()])
        project_reviewers = sorted([o.id for o in project.reviewer_users.all()])
        project_assignees = sorted([o.id for o in project.recruit_users.all()])

        self.assertEqual(card_assignees, project_assignees)
        self.assertEqual(card_reviewers, project_reviewers)

    @mock.patch("git_real.helpers.get_repo")
    @mock.patch.object(RecruitProject, "setup_repository")
    def test_start_project_creates_project_if_not_exists(self, get_repo, *_):

        card = factories.AgileCardFactory(
            status=AgileCard.READY,
            recruit_project=None,
            content_item=factories.ProjectContentItemFactory(
                project_submission_type=ContentItem.REPOSITORY
            ),
            assignees=[SocialProfileFactory().user],
        )

        self.assertIsNone(card.recruit_project)

        get_repo.return_value = {
            "full_name": "me/kiff",
            "owner": {"login": "me"},
            "ssh_url": "https://whatever.git",
            "private": True,
            "created_at": timezone.now().strftime(GITHUB_DATETIME_FORMAT),
            "archived": False,
        }

        card.start_project()

        self.assertIsNotNone(card.recruit_project)
        project = card.recruit_project

        card.refresh_from_db()
        project.refresh_from_db()

        self.assert_users_same(card=card, project=project)
        self.assertEqual(project.content_item, card.content_item)
        self.assertTrue(project.setup_repository.called)
        self.assertEqual(card.status, AgileCard.IN_PROGRESS)
        self.assertIsNotNone(project.start_time)
```

So maybe in your setUp you can already start the project and put it in review, which could make the rest of the tests a bit cleaner?